examples/pipelines/utils/multiply_data/submit.py

```diff
@@ -0,0 +1,222 @@
+"""Federated Learning Cross-Silo pipeline for artificially multiplying data to the silos' storages.
+This script:
+1) reads a config file in yaml specifying the number of silos and their parameters,
+2) reads the components from a given folder,
+3) builds a flexible pipeline depending on the config,
+4) configures each step of this pipeline to write to the right storage.
+"""
+import os
+import argparse
+import random
+import string
+import datetime
+import webbrowser
+import time
+import sys
+
+# Azure ML sdk v2 imports
+import azure
+from azure.identity import DefaultAzureCredential, InteractiveBrowserCredential
+from azure.ai.ml import MLClient, Input, Output
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.dsl import pipeline
+from azure.ai.ml import load_component
+
+# to handle yaml config easily
+from omegaconf import OmegaConf
+
+
+############################
+### CONFIGURE THE SCRIPT ###
+############################
+
+parser = argparse.ArgumentParser(description=__doc__)
+
+parser.add_argument(
+    "--config",
+    type=str,
+    required=False,
+    default=os.path.join(os.path.dirname(__file__), "config.yaml"),
+    help="path to a config yaml file",
+)
+parser.add_argument(
+    "--submit",
+    default=False,
+    action="store_true",
+    help="actually submits the experiment to AzureML",
+)
+
+parser.add_argument(
+    "--example",
+    required=True,
+    choices=["PNEUMONIA"],
```

Comment:
you don't actually need this parameter, we can use this pipeline to multiply any dataset with files.

Reply:
Removed!